src/cloudant/design_document.py

```diff
@@ -498,7 +498,12 @@ def fetch(self):
                         view_def.pop('reduce', None),
                         **view_def
                     )
-
+        # dbcopy is transformed to a new storage location, we need to de-transform
+        if 'options' in self:
+            if 'epi' in self['options']:
+                if 'dbcopy' in self['options']['epi']:
+                    for view in self['options']['epi']['dbcopy'].keys():
+                        self['views'][view]['dbcopy'] = self['options']['epi']['dbcopy'][view]
```

Comment:
@rhyshort so looking at this again my current change means there is no behaviour change from existing.

Adding dbcopy in a view:
`ddoc.add_view(view_name='myview', map_func='', reduce_func='', dbcopy='mycopy')`

This stores the dbcopy parameter in the view (which is the old Cloudant location) and it is accessible via: `ddoc['views']['myview']['dbcopy']`
If the ddoc is saved to older version of Cloudant it stays there. 
If it is saved to a CouchDB 2.0 version the **server** performs a transform that moves the dbcopy into the `{'options' : {'epi' : {'dbcopy' : {'myview' : 'mycopy'}}}` location.

When a design document is retrieved if a `dbcopy` parameter exists in the old location it remains there so is still accessible via `ddoc['views']['myview']['dbcopy']`
If the dbcopy parameter is in the new location the de-transform I added references it from the old location as well so it is accessible from both:
`ddoc['views']['myview']['dbcopy']`
and
`ddoc['options']['epi']['dbcopy']['myview']`

I think there is a slight hole with this approach in that when creating a view using the old style `dbcopy` kwarg it is not accessible in the new location until after a save/fetch with remote. However, it it still possible to add the `dbcopy` directly in the new location through the design document's dictionary. I feel like someone adding `dbcopy` in the old way wouldn't be looking for it in the new location anyway.

Does that make things more clear?
Are you suggesting that we should not make `ddoc['views']['myview']['dbcopy']` accessible unless specifically asked for?